src/keymap.cpp

```diff
@@ -471,6 +471,19 @@ void KeyMap::unset_key(const std::string& key, const std::string& context)
 	}
 }
 
+void KeyMap::unset_all_keys(const std::string& context)
+{
+	LOG(Level::DEBUG, "KeyMap::unset_all_keys() called");
```

Comment:
I'm not sure whether the log message should contain the `context` as well or not.

Reply:
Yes, I think it should.